src/test/resources/reference.conf

```diff
@@ -15,6 +15,7 @@ slick {
   port = 3306
   port = ${?MYSQL_PORT}
   db {
+    driver = "com.mysql.cj.jdbc.Driver"
```

Comment:
It's on the leonardo-configs branch for now, to help ensure that it doesn't go to prod.

Rawls uses older versions of Slick, Mysql, and Liquibase, so the format is slightly different.